javacutil/src/main/java/org/checkerframework/javacutil/Resolver.java

```diff
@@ -282,7 +282,7 @@ public Element findClass(String name, TreePath path) {
                             pck,
                             names.fromString(name),
                             Kinds.KindSelector.TYP);
-            if (res.getKind() == ElementKind.CLASS) {
+            if (res.getKind().isClass() || res.getKind().isInterface()) {
```

Comment:
This is fine, but `ElementUtils.isClassElement` would be shorter and easier to read.